### PR TITLE
[51] Send errors from CLI to Sentry

### PR DESCRIPTION
**Pull request by zipofar**

### CODE REVIEW

**Review by moklidia — APPROVED** [review]

File: .github/workflows/reusable.yaml [hunk 1]
```diff
@@ -1,5 +1,8 @@
 name: Reusable Uffizzi Preview Workflow
 
+env:
+  LOGGER_KEY: "aHR0cHM6Ly80NGI3Zjk5MTFiODU0NWI5YTMzMDY4NzRhY2ZjYjJjOUBvMzI0\nMzExLmluZ2VzdC5zZW50cnkuaW8vNDUwNDM2MTIxMjI0ODA2NA==\n"
```
Comment (axisofentropy) [inline .github/workflows/reusable.yaml:4]: I don't think that `\n` should be there.

File: .github/workflows/reusable.yaml [hunk 2]
```diff
@@ -230,7 +236,7 @@ jobs:
       - name: Deploy New Preview
         id: create-preview
         if: ${{ env.DEPLOYMENT_ID == '' && inputs.compose-file-cache-key != '' }}
-        uses: UffizziCloud/preview-action@v2.5.0
+        uses: UffizziCloud/preview-action@v2
```
Comment (axisofentropy) [inline .github/workflows/reusable.yaml:239]: :cyclone:

File: .github/workflows/reusable.yaml [hunk 3]
```diff
@@ -179,7 +184,7 @@ jobs:
 
       - name: Delete Preview from Uffizzi
         if: ${{ inputs.compose-file-cache-key == '' }}
-        uses: UffizziCloud/delete-preview-action@v1
+        uses: UffizziCloud/delete-preview-action@feature/7_send-errors-from-cli-to-sentry
```
Comment (zipofar) [inline .github/workflows/reusable.yaml:187]: Do not forget change

File: .github/workflows/reusable.yaml [hunk 4]
```diff
@@ -230,7 +236,7 @@ jobs:
       - name: Deploy New Preview
         id: create-preview
         if: ${{ env.DEPLOYMENT_ID == '' && inputs.compose-file-cache-key != '' }}
-        uses: UffizziCloud/preview-action@v2.5.0
+        uses: UffizziCloud/preview-action@feature/51_send-errors-from-cli-to-sentry
```
Comment (zipofar) [inline .github/workflows/reusable.yaml:239]: Do not forget change

File: .github/workflows/reusable.yaml [hunk 5]
```diff
@@ -274,7 +281,7 @@ jobs:
           args: --basic --user ${{ secrets.url-username }}:${{ secrets.url-password }} --retry 12 --retry-all-errors --retry-delay 0 --fail --location ${{ steps.create-preview.outputs.url }}${{ inputs.healthcheck-url-path }}
 
       - name: Update Existing Preview
-        uses: UffizziCloud/update-preview-action@v1
+        uses: UffizziCloud/update-preview-action@feature/8_send-errors-from-cli-to-sentry
```
Comment (zipofar) [inline .github/workflows/reusable.yaml:284]: Do not forget change

File: action.yaml [hunk 6]
```diff
@@ -95,7 +98,7 @@ outputs:
     description: 'Expiration as a UNIX timestamp'
 runs:
   using: 'docker'
-  image: 'docker://uffizzi/cli:v1'
+  image: 'docker://uffizzi/cli:pr-201'
```
Comment (zipofar) [inline action.yaml:101]: Do not forget change

File: .github/workflows/reusable.yaml [hunk 7]
```diff
@@ -155,7 +158,7 @@ jobs:
       # If PR event, look for an existing Preview Deployment.
       - name: Find Deployment for this Pull Request
         id: find-deployment
-        uses: docker://uffizzi/cli:v1
+        uses: docker://uffizzi/cli:pr-201
```
Comment (zipofar) [inline .github/workflows/reusable.yaml:161]: Do not forget change to v1

**Pull request merged**